dist/profile/templates/jenkinscontroller/prompt.sh.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+export PROMPT_COMMAND='PS1="${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@<%= @ci_fqdn %>$\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$"'
```

Comment:
```suggestion
export PROMPT_COMMAND='PS1="${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@<%= @ci_fqdn %>\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ "'
```

Reply:
Tested on ci.jenkins.io manually:

- Removing the first `$`
- Adding a space at the end